index.html

```diff
@@ -45,6 +45,155 @@ <h2 class="Introduction">I am a Software Developer</h2>
         don't hesitate to contact me.</p>
 </section>
```

Comment:
- [ ] I am not sure what could have happened but it looks like the initial part of the project you first created had been compromised. From the image below you can see that everything looks displaced. Kindly ensure to solve all of this before you resubmit for review :wink:
- Your design now
![image](https://user-images.githubusercontent.com/102699958/213861049-dad97d4d-6e92-45f9-8999-7714a2154594.png)

Reply:
I don't get it but I have pushed every code in my file and I have not changed a single thing from my previous project